### PR TITLE
Create 2023-07-10-Arcadia-Finance.md

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, fix the article according to a changes requested in your other PR's.

File: content/attacks/posts/2023-07-10-Arcadia-Finance.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+date: 2023-07-10
+target-entities: Arcadia Finance
+entity-types:
+  - DeFi
```
Comment (JediFaust) [inline content/attacks/posts/2023-07-10-Arcadia-Finance.md:5]: Add `Lending Platform`

File: content/attacks/posts/2023-07-10-Arcadia-Finance.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   - 11 ETH (20,558 USD)
+
+## Timeline
+
+- **July 10, 2023, 01:16:07 AM UTC:** The [first malicious transaction occurred](https://optimistic.etherscan.io/tx/0xca7c1a0fde444e1a68a8c2b8ae3fb76ec384d1f7ae9a50d26f8bfdd37c7a0afe).
+- **July 10, 2023, 01:21:59 AM UTC:** A [second malicious transaction occurred](https://etherscan.io/tx/0xefc4ac015069fdf9946997be0459db44c0491221159220be782454c32ec2d651).
+- **July 10, 2023, 10:42 AM UTC:** Suspicious transactions were detected by [PeckShield](https://twitter.com/PeckShieldAlert/status/1678248292327763968)   
+- **July 10, 2023, 01:10 PM UTC:** The Arcadia Finance team [announced](https://twitter.com/ArcadiaFi/status/1678285634727706625) the hack on Twitter.
+- **July 10, 2023:** Immunebytes [published](https://www.immunebytes.com/blog/arcadia-finance-exploit-detailed-hack-analysis/) a detailed analysis of the incident.
+- **July 10, 2023:** Arcadia Finance [published exploit Post-Mortem](https://arcadiafinance.medium.com/post-mortem-72e9d24a79b0)
```
Comment (JediFaust) [inline content/attacks/posts/2023-07-10-Arcadia-Finance.md:44]: Merge this events, since the timestamp is the same.

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
@svg-arch please pay attention to the editor's notes from the bot

File: content/attacks/posts/2023-07-10-Arcadia-Finance.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Hacker Wallets:
+
+- [0xd3641c912a6a4c30338787e3c464420b561a9467](https://optimistic.etherscan.io/address/0xd3641c912a6a4c30338787e3c464420b561a9467)
+- [0x5c75e94dd0ab9c10bfd1b8073dafef031d3c050d](https://etherscan.io/address/0x5c75e94dd0ab9c10bfd1b8073dafef031d3c050d)
+
+## Losses
+
+The total loss from the Acradia Finance hack amounted to approximately $455,00, distributed across the following networks:
+
+- [Ethereum](https://phalcon.blocksec.com/explorer/tx/eth/0xefc4ac015069fdf9946997be0459db44c0491221159220be782454c32ec2d651):
```
Comment (JediFaust) [inline content/attacks/posts/2023-07-10-Arcadia-Finance.md:29]: Don't need to duplicate links already provided in the article. In the timeline section there are both transactions.

File: content/attacks/posts/2023-07-10-Arcadia-Finance.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+   - 148 ETH (275,843 USD)
+   - 103,200 USDC
+
+- [Optimism](https://phalcon.blocksec.com/explorer/tx/optimism/0xca7c1a0fde444e1a68a8c2b8ae3fb76ec384d1f7ae9a50d26f8bfdd37c7a0afe):
```
Comment (JediFaust) [inline content/attacks/posts/2023-07-10-Arcadia-Finance.md:33]: Same here.

File: content/attacks/posts/2023-07-10-Arcadia-Finance.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- **July 10, 2023:** Immunebytes [published](https://www.immunebytes.com/blog/arcadia-finance-exploit-detailed-hack-analysis/) a detailed analysis of the incident. Arcadia Finance [published exploit Post-Mortem](https://arcadiafinance.medium.com/post-mortem-72e9d24a79b0).
+- **July 25, 2023:** Arcadia is [pausing](https://arcadiafinance.medium.com/sunsetting-arcadia-v1-89a5d03bd8cc) actions to focus on user fund recovery and transitioning to a more secure Arcadia V2.
+- **October 18, 2023:** Arcadia [presented](https://arcadiafinance.medium.com/introducing-arcadia-v2-79ee345af20b) version 2 of the protocol.
+
+## Security Failure Causes
+
+- **Smart Contract Vulnerability:** The Arcadia Finance hack was caused by vulnerabilities in its smart contracts, particularly in functions related to vault management and liquidation. These issues were exacerbated by inadequate reentrancy protection and poor validation of external inputs. The attacker bypassed security checks to redirect assets and execute unauthorized transactions, resulting in significant fund loss.
```
Comment (JediFaust) [inline content/attacks/posts/2023-07-10-Arcadia-Finance.md:49]: There are claims that it was Reentrancy Issue, if it is, please add 'Reentrancy Attack' to the attack types section.

**Review by JediFaust — DISMISSED** [review]

**Comment by evgenydmitriev** [comment]
@svg-arch, please pay attention to numbers and inconsistencies in names. For example, my spellchecker caught the A**cr**adia typo. In addition to this, I would advise using GPT4, Gemini, or Claude 3 models for double-checking your submissions before requesting a pr review. A prompt like "help me find inconsistencies in the article below" could probably address most of the problems.

**Pull request merged**